express-mung/express-mung.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for express-mung 0.4.2
+// Project: https://github.com/richardschneider/express-mung
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../express/express.d.ts"/>
+/// <reference path="../node/node.d.ts"/>
+
+declare module "express-mung" {
+    import { Request, Response } from "express";
+    import * as http from "http";
+
+    type Transform = (body: Object, request: Request, response: Response) => Object;
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html
> If you’re tempted to use the type Object, consider using any instead. There is currently no way in TypeScript to specify an object that is “not a primitive”.